src/MongoDB.Driver/IMongoCollection.cs

```diff
@@ -224,6 +224,7 @@ public interface IMongoCollection<TDocument>
 
         /// <summary>
         /// Counts the number of documents in the collection.
+        /// For a fast count of the total documents in a collection see <see cref="EstimatedDocumentCount"/>.
```

Comment:
There are more than 2 `CountDocuments` overloads. Including `IMongoCollectionExtensions` methods.

Reply:
You are correct! I've added docstring to the missing overloads and to the extension methods